doc/policy/mempool-replacements.md

```diff
@@ -63,6 +58,8 @@ This set of rules is similar but distinct from BIP125.
 
 ## History
 
+* Signaling for replace-by-fee is no longer required as of [PR 30592](https://github.com/bitcoin/bitcoin/pull/30592).
```

Comment:
Move to bottom to retain chronological ordering?

Reply:
done